README.md

```diff
@@ -11,20 +11,22 @@
 ## Contents
   		  
 A curated list of awesome Liferay 7.x and DXP docs, libraries, blogs and other resources.
-- [Awesome Liferay](#awesome-liferay)
-    - [Getting started](#gettingstarted)
-    - [Documentation](#documentation)
-    - [Products](#products)
-        + [Platform](#platform)
-        + [Frameworks](#frameworks)
-        + [Mobile](#mobile)
-        + [Cloud](#cloud)
-    - [Samples](#samples)
-        + [Frontend](#frontend-samples)
-        + [Tips and Tricks](#tips-and-tricks)
-    - [Tools](#tools)
-    - [Compatibility Matrix](#compatibility-matrix)
-    - [Blogs](#blogs-and-community)
+- [Awesome Liferay ![Awesome](https://github.com/sindresorhus/awesome)](#awesome-liferay-awesomehttpsgithubcomsindresorhusawesome)
```

Comment:
Hi @yasuflatland-lf .... can you please check this line? The link seems to be wrong